src/auditlog/models.py

```diff
@@ -110,6 +116,7 @@ class Action:
     changes = models.TextField(blank=True, verbose_name=_("change message"))
     actor = models.ForeignKey(settings.AUTH_USER_MODEL, blank=True, null=True, on_delete=models.SET_NULL, related_name='+', verbose_name=_("actor"))
     timestamp = models.DateTimeField(auto_now_add=True, verbose_name=_("timestamp"))
+    additional_data = JSONField(blank=True, null=True)
```

Comment:
A verbose name (for example ```_("additional data")```) should be added to the field definition to make translation possible.